src/analytics/Properties.tsx

```diff
@@ -1116,8 +1116,13 @@ interface FiatExchangeEventsProperties {
     provider: string
     flow: CICOFlow
   }
+  [FiatExchangeEvents.cico_fc_transfer_error]: {
+    error?: string
+    provider: string
+    flow: CICOFlow
+  }
   [FiatExchangeEvents.cico_fc_transfer_success]: {
-    txHash?: string
+    txHash: string | null
```

Comment:
Not directly related to this PR, but we have other events from the status screen that also optionally take txHash (`cico_fc_transfer_success_complete` and `cico_fc_transfer_success_view_tx`). Should we change the type for those as well?

Reply:
fixed